commonly_proposed.md

```diff
@@ -2,8 +2,6 @@
 
 This is a list of changes to the Swift language that are frequently proposed but that are unlikely to be accepted.  If you're interested in pursuing something in this space, please familiarize yourself with the discussions that we have already had.  In order to bring one of these topics up, you'll be expected to add new information to the discussion, not just to say, "I really want this" or "this exists in some other language and I liked it there".
 
-Additionally, proposals for out-of-scope changes will not be scheduled for review. The [readme file](README.md) identifies a list of priorities for the next major release of Swift, and the [proposal review status page](https://apple.github.io/swift-evolution/) includes a list of changes that have been deferred for future discussion because they were deemed to be out of scope at the time of review (in addition to a list of changes proposed and rejected after a formal review).
```

Comment:
Why was this line deleted as part of this PR?  We should update the link to point at the new SE dashboard.

Reply:
Seems like a more narrowly scoped change would be to delete just the last clause (everything after the comma); it is part of our responsibility as the language workgroup to continue to describe priorities periodically for upcoming releases with core team review.